libs/request.js

```diff
@@ -49,6 +49,8 @@ module.exports = function (options = {}) {
 
   options = Object.assign(defaultOptions, options)
 
+  if (options.cheerio === true && options.json === undefined) options.json = false
```

Comment:
Isn't `if (options.cheerio && !options.json)` a better test ?

Reply:
You're right